tests/TestHelpers/GraphHelper.php

```diff
@@ -31,7 +31,7 @@ private static function getRequestHeaders(): array {
 	 * @return string
 	 */
 	public static function getUUIDv4Regex(): string {
-		return '[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}';
+		return '[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-4[0-9A-Fa-f]{3}-[89ABab][0-9A-Fa-f]{3}-[0-9A-Fa-f]{12}';
 	}
```

Comment:
I see these changes in 3 of your PR's, if the PR's are branched out from one another and there's dependency better to keep the PR in draft until the main PR is merged.